hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/constants/AbfsOperationConstants.java

```diff
@@ -0,0 +1,28 @@
+package org.apache.hadoop.fs.azurebfs.constants;
+
+public final class AbfsOperationConstants {
+    public static final String ACCESS = "AS";
+    public static final String APPEND = "AP";
+    public static final String CREATE = "CR";
+    public static final String CREATEFILESYSTEM = "CF";
+    public static final String DELETE = "DL";
+    public static final String GETACLSTATUS = "GS";
+    public static final String GETATTR = "GA";
+    public static final String GETFILESTATUS = "FS";
+    public static final String LISTSTATUS = "LS";
+    public static final String MKDIR = "MK";
+    public static final String MODIFYACL = "MA";
+    public static final String OPEN = "OP";
+    public static final String PATH = "PA";
+    public static final String PERMISSION = "PR";
+    public static final String READ = "RD";
+    public static final String REMOVEACL = "RA";
+    public static final String REMOVEACLENTRIES = "RE";
+    public static final String REMOVEDEFAULTACL = "DA";
+    public static final String RENAME = "RN";
+    public static final String SETATTR = "ST";
+    public static final String SETOWNER = "SO";
+    public static final String SETACL = "SA";
+    public static final String TESTOP = "TS";
```

Comment:
Why is this needed ?

Reply:
Generic op name for the AbstractAbfsIntegrationTest method that creates tracingContext instance for tests that require TC only for compilation